riker/api.py

```diff
@@ -16,7 +16,7 @@
 from boto.ec2.autoscale import LaunchConfiguration
 from boto.ec2.autoscale import AutoScalingGroup
 from boto.ec2.elb.attributes import ConnectionDrainingAttribute
-from fabric.api import task, run, local, env, sudo, lcd, execute, put
+from fabric.api import task, run, local, env, sudo, lcd, execute, put, settings
```

Comment:
I need to remove the changes on this line.  I didn't end up using the settings from fabric.api.